ifopt_core/include/ifopt/constraint_set.h

```diff
@@ -126,10 +126,10 @@ class ConstraintSet : public Component {
    * or shorthands to specific variable sets want to be saved for quicker
    * access later. This function can be overwritten for that.
    */
-  virtual void InitVariableDependedQuantities(const VariablesPtr& x_init){};
```

Comment:
Replace `x_init` with `/*x_init*/` and that will solve the issue so no need to add `(void) x_init`.